src/components/app/details/triggerView/workflow/Workflow.tsx

```diff
@@ -5,21 +5,35 @@ import { TriggerExternalCINode } from './nodes/TriggerExternalCINode'
 import { TriggerLinkedCINode } from './nodes/TriggerLinkedCINode'
 import { TriggerCDNode } from './nodes/triggerCDNode'
 import { TriggerPrePostCDNode } from './nodes/triggerPrePostCDNode'
-import { RectangularEdge as Edge } from '../../../../common'
+import { getCIPipelineURL, RectangularEdge as Edge } from '../../../../common'
 import { WorkflowProps, NodeAttr, PipelineType, WorkflowNodeType } from '../types'
 import { WebhookNode } from '../../../../workflowEditor/nodes/WebhookNode'
 import DeprecatedPipelineWarning from '../../../../workflowEditor/DeprecatedPipelineWarning'
+import { GIT_BRANCH_NOT_CONFIGURED } from '../../../../../config'
 
 export class Workflow extends Component<WorkflowProps> {
+    goToWorkFlowEditor = (node: NodeAttr) => {
+        if (node.branch === GIT_BRANCH_NOT_CONFIGURED) {
+            this.props.history.push(
+                getCIPipelineURL(
+                    this.props.match.params.appId.toString(),
+                    this.props.id.toString(),
+                    node.downstreams[0].split('-')[1].toString(),
```

Comment:
add fail-safe checks if required

Reply:
We don't need fail safe check here as node.downstreams always contains a value, for eg;- 'CI-255'